topydo/lib/Colors.py

```diff
@@ -58,7 +58,13 @@ def normalize_color(p_input):
         try:
             return color_names_dict[p_input]
         except KeyError:
-            return p_input
+            try:
```

Comment:
I believe we need 3 types of values returned here:

1. appropriate color id (valid config or default)
2. `-1` - NEUTRAL_COLOR (invalid config)
3. something that will result in not applying color at all (empty config value)

In my code above this idea is (poorly) implemented with third type of returned value which as string (raw `p_input`).